src/messaging/ReliableMessageContext.cpp

```diff
@@ -242,21 +215,18 @@ CHIP_ERROR ReliableMessageContext::HandleNeedsAckInner(uint32_t messageCounter,
     {
         if (IsAckPending())
         {
-#if !defined(NDEBUG)
             ChipLogDetail(ExchangeManager,
                           "Pending ack queue full; forcing tx of solitary ack for MessageCounter:" ChipLogFormatMessageCounter
                           " on exchange " ChipLogFormatExchange,
                           mPendingPeerAckMessageCounter, ChipLogValueExchange(GetExchangeContext()));
-#endif
             // Send the Ack for the currently pending Ack in a SecureChannel::StandaloneAck message.
             ReturnErrorOnFailure(SendStandaloneAckMessage());
         }
 
         // Replace the Pending ack message counter.
         SetPendingPeerAckMessageCounter(messageCounter);
-        mNextAckTimeTick = static_cast<uint16_t>(
-            CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT_TICK +
-            GetReliableMessageMgr()->GetTickCounterFromTimeDelta(System::SystemClock().GetMonotonicTimestamp()));
+        using namespace System::Clock::Literals;
```

Comment:
Where do we use Literals here?

Reply:
It is used inside `CHIP_CONFIG_RMP_DEFAULT_ACK_TIMEOUT`, it is a C style define using user-defined literal.